### DISCUSSION TITLE
Requesting a minor release

### DISCUSSION THREAD

**Discussion by antimora** [starkat99/half-rs#87] [General]
@starkat99 Thank you for fixing the issue https://github.com/starkat99/half-rs/issues/84. Could you please release the fix with a minor version? We are want to include a new feature for our upcoming release of [Burn](https://github.com/burn-rs/burn)

(CCing @nathanielsimard)

**Comment by VoidStarKat** [comment]
I'll try to get to it this weekend. Been very busy recently

**Reply by antimora** [reply]
Thank you for your work! Hoping we can get a minor update release soon. Let us know how we could help if needed.

**Comment by antimora** [comment]
Hi @starkat99. Hope you're doing well. Do you think we could the release out soon?

**Comment by VoidStarKat** [comment]
Sorry it took so long, lots of IRL issues to deal with, but 2.3.0 is now released.

**Reply by antimora** [reply]
YaY. Thank you!
